dkist/dataset/tests/test_dataset.py

```diff
@@ -51,13 +52,15 @@ def test_repr(dataset, dataset_3d):
 
 
 def test_wcs_roundtrip(dataset):
+    # TOO generalise this so mark.accept_cli_dataset will work, if possible
     p = (10*u.pixel, 10*u.pixel)
```

Comment:
This should do the trick:
```suggestion
    p = [1*u.pixel] * dataset.wcs.pixel_n_dim
```

Reply:
That makes sense but then do we still need to have a 2d and 3d version of this test? Or shall we just lump them together with this change?

Reply:
Oh you've already commented on that. Is the other one not doing the same as this but in 3d?

Reply:
I don't know why the other test splits the pixel asserts, we can try it and see though.

Reply:
Ah I assume it's because the third dimension has different dimensions, but we can probably account for that as well.

Reply:
> This should do the trick:

Turns out it does not because the order `pixel_to_world` gives coords in is not necessarily the order `world_to_pixel` needs to take them in. This probably can be generalised and it may not even be too hard but I think it's not worth it blocking this any further right now.

Reply:
errr, wat? I think that is a bug

Reply:
For the record, per conversation offline yes this is a bug but it's a bug in the gwcs and not in the tests, so the test can stay as it is, since catching these bugs is exactly the point.